multipass/how-to/how-to-set-up-a-graphical-interface.md

```diff
@@ -15,35 +15,62 @@ The graphical desktop can be viewed in various ways. In this document, we descri
 
 The images used by Multipass do not come with a graphical desktop installed. For this reason, a desktop environment must be installed (here we use `ubuntu-desktop` but there are as many other options as flavors of Ubuntu exist), along with the RDP server (we will use `xrdp` but there are also other options such as `freerdp`). 
 
-To do this, first you need to log in to the running Multipass instance:
+To do this, you need to log into a running Multipass instance. Start by listing your instances:
 
 ```plain
-$ multipass shell headbanging-squid
+multipass list
+```
+
+Sample output:
+
+```plain
+Name                    State             IPv4             Image
+headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
+```
+
+Next open a shell into the running instance:
+
+```plain
+multipass shell headbanging-squid
 ```
 
 Once inside the instance, run the following commands to install `ubuntu-desktop` and `xrdp`:
 
 ```plain
-$ sudo apt update
-$ sudo apt install ubuntu-desktop xrdp
+sudo apt update
+sudo apt install ubuntu-desktop xrdp
 ```
 
 Then, we need a user with a password in order to log in. One possibility is setting a password for the default `ubuntu` user:
 
 ```plain
-$ sudo passwd ubuntu
+sudo passwd ubuntu
 ```
 
 You will be asked to enter and re-enter a password. 
 
 You are done on the server side!
 
-Now, quit the Ubuntu shell on the instance with the `logout` command and find out in the host the IP address to connect to:
+Now, quit the Ubuntu shell on the running instance with the `exit` command, and take note of the IP address to connect to. The instance's IP address can be found in the output of `multipass list` from the first step above, or you can use the `multipass info` command as well.
 
 ```plain
-$ multipass list
-Name                    State             IPv4             Image
-headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
+multipass info headbanging-squid
+```
+
+Sample output:
+
+```plain
+Name:           headbanging-squid
+State:          Running
+Snapshots:      0
+IPv4:           10.49.93.209
+Release:        Ubuntu 24.04 LTS
```

Comment:
Here as well: 

```suggestion
Release:        Ubuntu 22.04 LTS
```

Reply:
@DanRoscigno check the LTS version used in the example, it should be 22.04 for `headbanging-squid`